storm-core/src/clj/org/apache/storm/daemon/common.clj

```diff
@@ -86,26 +40,6 @@
 (defn new-executor-stats []
   (ExecutorStats. 0 0 0 0 0))
```

Comment:
Why are we not moving ExecutorStats, SupervisorInfo, StormBase and Assignment to java?

I know we have thrift equivalents of these already.  Is the plan to move them to java as we change the daemon code to use java?  If so I am fine with it, I just want to understand for sure what is happening.